### PR TITLE
First round of icon updates.

### PR DESCRIPTION
**Pull request by pixlwave**
First part of #2002. I'll tackle the Settings List separately as it needs more work. Also updates Compound too.

| User Menu | Room Attachments Menu | Reply Views |
| - | - | - |
| Screenshot 2023-11-13 at 11 16 31 am | Screenshot 2023-11-13 at 4 15 16 pm | Screenshot 2023-11-13 at 12 33 30 pm |

| Room Context Menu | User Details Edit Screen |
| - | - |
| Simulator Screenshot - iPhone 14 - 2023-11-13 at 16 16 03 | Simulator Screenshot - iPhone 14 - 2023-11-13 at 16 17 01 |

### CODE REVIEW

File: UnitTests/__Snapshots__/PreviewTests/test_appLockSettingsScreen.Face-ID.png
Comment (Velin92) [inline UnitTests/__Snapshots__/PreviewTests/test_appLockSettingsScreen.Face-ID.png:1]: Why have these been removed? They have been renamed?

File: ElementX/Resources/Assets.xcassets/images/circle.imageset/circle.svg
Comment (Velin92) [inline ElementX/Resources/Assets.xcassets/images/circle.imageset/circle.svg:1]: Why these ares till in the asset catalogue? they have not been moved yet to Compound?

**Review by Velin92 — APPROVED** [review]
LGTM, just some questions

File: ElementX/Resources/Assets.xcassets/images/circle.imageset/circle.svg
Comment (pixlwave) [reply ElementX/Resources/Assets.xcassets/images/circle.imageset/circle.svg:1]: Nope, they're waiting for review. I've asked that we don't wait for review in the future and instead move faster and fix in Compound directly, so hopefully after the second part of #2002, we'll be able to just remove the the catalogue and not add new things.

**Pull request merged**